packages/compass/src/setup-hadron-distribution.ts

```diff
@@ -33,6 +33,15 @@ if (
   process.type === 'browser'
 ) {
   if (process.platform === 'linux') {
+    const isLibsecretInstalled = () => {
+      try {
+        process.dlopen({ exports: {} }, 'libsecret-1.so.0');
+        return true;
+      } catch (err: any) {
+        return err && err.message?.includes('did not self-register');
+      }
+    };
```

Comment:
Totally forgot about this PR, @addaleax does this check looks good to you? The error codes between not installed and failed to open look the same, so I'm checking for the error message here, not sure if there's a better way to do it?